andela/mag-ah-frontend
# #159965262 Install and setup sass pre-processing.

#### What does this PR do?
 * Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client

#### Description of Task to be completed?
 * Change App.css file to App.sccs

#### How should this be manually tested?
- Clone this repository: https://github.com/andela/mag-ah-frontend.git 
- In your terminal run `git clone https://github.com/andela/mag-ah-frontend.git`.
- Navigate to `mag-ah-frontend` folder .
- Check out to branch `ch-setup-sass-#159965262`
- Run `npm install` to install app dependencies
- Run `npm start` to run the application 

#### What are the pivotal tracker story for the task
[Set up sass](https://www.pivotaltracker.com/n/projects/2192364/stories/159965262)
